notebooks/scarlet-measure.ipynb

```diff
@@ -3,10 +3,19 @@
   {
```

Comment:
<p>Did you mean to remove the output of this cell? either it doesn't matter users can run the notebook and get the output, just wanted to check if this was your intention. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSSTDESC/BlendingToolKit/pull/162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>I must have forgotten to run it again ; I also have to add the interactive=True back so I will do it now.</p>